src/components/Disputes/DisputeCurrentRuling.js

```diff
@@ -116,14 +117,15 @@ function DisputeCurrentRuling({ dispute }) {
                       margin-right: ${0.5 * GU}px;
                     `}
                   >
-                    Your voting weight
+                    {showMyWeight ? 'Your v' : 'V'}oting weight
                   </span>
-                  <Help>
-                    The same juror can be drafted multiple times to arbitrate
-                    the same dispute for the same round. When that happens,
-                    their voting weight will be proportional to the number of
-                    times is drafted, as well as the % of ANJ locked in the
-                    Active balance.
+                  <Help hint="What's the voting weight?">
```

Comment:
```suggestion
                  <Help hint="What's the voting weight?">
```

Maybe we don't need the `hint`; it doesn't really summarize what we explain below. If we wanted it, it'd be something like "Why do some jurors have more voting weight?", but this is not something a natural question from the information we currently show (we don't breakdown the voting weights at all, I think?).

Reply:
Good point. We don't break them down no, at least for now.

Will leave it `''`  since it's a required prop by the `<Help />`